inference_engines/x-inference-engines/x_perf_monitor.h

```diff
@@ -0,0 +1,50 @@
+/** Copyright 2019 Xperi Corporation.
```

Comment:
Yes, by signing the CLA, all contributors are agreeing to that and although of course anything you write is by definition copyright by the author, the controlling agreement is the CLA and the license that Glow is provided under. So, yes, in order to avoid confusion, all copyright headers should match what we are using in the rest of Glow. Thanks Jack for making sure that's the case--and thank you very kindly @wyessen for your contribution(s)!

Reply:
Thanks @jackm321 for the review! I'll be getting to the technical points you made shortly.

@likethesky On legal matters: our contribution is of course provided under the same license as Glow, and we did sign the CLA. The only thing that is different in the headers is the "Copyright 2019 Xperi Corporation" rather than "Copyright ... Facebook...". 

So my question is whether we should change our header only to avoid confusion, or there is legal pressure? Just trying to gather enough info before I kick this over to the relevant parties on our end. Thanks!

Reply:
I checked w/legal before replying the first time. So yes, some legal oversight has occurred and that's their rec. Of course, the Apache 2.0 license allows anyone to copy/use any of this code at any time, so you guys are protected. Also, I probably misspoke or wasn't totally clear in my statement above, when you "write" anything, it's copyright you, but once you contribute it under our CLA, the copyright is granted to Facebook. See under "2. Grant of Copyright License" here: https://code.facebook.com/cla/

Hope that helps. So yes, needs to be (c) Facebook, please use the standard copyright used throughout Glow (and I believe every other of the thousand+ OSS projects we have). Cheers!